doc/docs/overview.md

```diff
@@ -14,7 +14,7 @@ Opens the drop-down with the list of devices connected to the computer. You can
 
 When you select a device, the following actions become available in the side panel:
 
-- **Write config** - Writes the selected configurations to the development kit.
+- **Write config** - Writes the selected configurations to the development kit by overwriting the defaults from the board definition file.
```

Comment:
W: I would simply say "Writes the selected configurations to the development kit, overwriting the defaults." I'm not sure the user is supposed to care about the board definition files, but it might be.